client/my-sites/plugins/plugin-details.jsx

```diff
@@ -250,10 +248,11 @@ function PluginDetails( props ) {
 
 	// We need to get the breadcrumbs here, after initial append dispatches on server.
 	const breadcrumbs = useSelector( getBreadcrumbs );
+	const [ initialBreadcrumbs ] = useState( breadcrumbs );
 
 	useEffect( () => {
-		setBreadcrumbs( breadcrumbs.length !== 0 );
-	}, [ setBreadcrumbs, breadcrumbs.length ] );
+		setBreadcrumbs( initialBreadcrumbs );
+	}, [ setBreadcrumbs ] );
```

Comment:
I intentionally left it out because `initialBreadcrumbs` should never change; it can contain the previous page's breadcrumbs to which we append the plugin name.